app/src/main/java/org/p2p/wallet/auth/repository/CountryInMemoryRepository.kt

```diff
@@ -0,0 +1,24 @@
+package org.p2p.wallet.auth.repository
+
+import org.p2p.wallet.auth.model.CountryCode
+
+class CountryInMemoryRepository(
+    private val countryCodeRepository: CountryCodeLocalRepository,
+) : CountryRepository {
+    override suspend fun getAllCountries(): List<Country> {
+        return countryCodeRepository.getCountryCodes()
+            .map { it.extractCountry() }
+    }
+
+    override suspend fun detectCountryOrDefault(): Country {
+        val countryCodeByDetekt = countryCodeRepository.detectCountryCodeBySimCard()
```

Comment:
maybe `detectedCode`? and there's a typo: countryCodeByDete[c]t

Reply:
I am full in Detekt, thanks